src/coreclr/jit/morph.cpp

```diff
@@ -6582,6 +6582,18 @@ void Compiler::fgMorphCallInline(GenTreeCall* call, InlineResult* inlineResult)
             fgMorphStmt->SetRootNode(gtNewNothingNode());
         }
     }
+#ifdef DEBUG
+    else if (call->gtCallType == CT_USER_FUNC)
+    {
+        FILE* logFile;
+        LPCWSTR logFileName = JitConfig.JitListSuccessfulInlinesToFile();
+        if ((logFileName != nullptr) && ((logFile = _wfopen(logFileName, W("a"))) != nullptr))
+        {
+            fprintf(logFile , "%s  <--  %s\n", info.compFullName, eeGetMethodFullName(call->gtCallMethHnd));
+            fclose(logFile);
+        }
+    }
+#endif
```

Comment:
I know that we can extract that information from the Dump/ETW but it's just easier for now to quickly diff two runtimes. I'll most likely delete it later.